.travis.yml

```diff
@@ -53,8 +53,17 @@ jobs:
     - os: windows
 
   include:
+    - stage: Lint
```

Comment:
I don't think linting should be a separate stage. If we run this in the test stage, it'll run in parallel with everything else. But in its own stage, it will delay the other tests from even starting until it has completed. I know black is fast, but Travis still has to spin up the vm and clone the repo, etc, so there's can be up to a minute delay waiting for this.

Reply:
I've changed it. One advantage of a "Linting" stage is it can "fail fast" to let people know that (hopefully minor) changes are needed.

Reply:
Yup, that's definitely an advantage, but that still happens even if the jobs are being run in parallel (the build will fail with the first failed job). The rest of the jobs will still run, but that can also be useful in case there's more than 1 kind of problem with the build.